README.md

```diff
@@ -48,13 +48,13 @@ uses the raw configurations to do its magic.
 
 In the sbt configuration file:
 
-use scala `2.10` or `2.11`:
+use scala `2.10`, `2.11` or `2.12`:
 
 ```scala
 scalaVersion := "2.11.8" // or "2.10.5"
```

Comment:
Nitpick: 2.12 could appear on this line too.

Alternatively, I think you could probably just drop this section, as it seems:
- A given that a `scalaVersion` need to be specified.
- overspecific insofar as pureconfig doesn't have a dependency on a specific patch version of scala.